extensions/functions/setup.py

```diff
@@ -12,13 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from setuptools import find_packages, setup
+import os
 
-from opencensus.azure.extensions import __version__
+from setuptools import find_namespace_packages, setup
+
+BASE_DIR = os.path.dirname(__file__)
+VERSION_FILENAME = os.path.join(BASE_DIR, "src", "opencensus", "azure", "functions", "version.py")
+PACKAGE_INFO = {}
+with open(VERSION_FILENAME) as f:
+    exec(f.read(), PACKAGE_INFO)
 
 setup(
-    name='opencensus-extensions-azure',
-    version=__version__,  # noqa
+    name='opencensus-extensions-azure-functions',
```

Comment:
Should this be `opencensus-azure-functions-extension`? Since the import statement is `from opencensus.azure.functions.extension import OpenCensusExtension`.